env/test/test.properties

```diff
@@ -2,4 +2,5 @@ CHANNEL = ocp-4.6
 OPERATOR_ENV = prod
 CSV_VERSION = v1.0.1
 INSTALL_OPERATOR = no
-SECRET_TOKEN = 1234567
\ No newline at end of file
+SECRET_TOKEN = 1234567
+CATALOG_SOURCE = pre-stage-operators
```

Comment:
When/where is this value used?

Reply:
`pkg/config/config.go` mentions `redhat-operators` but that is definitely not a default value and actually, I would prefer it to be default

Reply:
sorry it's typo in `pkg/config/config.go` usually it gets value by default as `pre-stage-operators` but user can explicitly specify it's value at runtime if he want to use `redhat-operators`

my question here is what to default here??
`CATALOG_SOURCE = pre-stage-operators or redhat-operators`

mostly as we are using release-tests to test against pre-stage build it's safe to assume default as `pre-stage-operators`

Reply:
OK, let's leave it as is for now, we can re-evaluate later.